test/unit/org/apache/cassandra/index/sai/cql/StorageAttachedIndexDDLTest.java

```diff
@@ -263,6 +263,120 @@ public void shouldCreateIndexIfExists()
         assertEquals(1, saiCreationCounter.get());
     }
 
+    @Test
+    public void shouldBeCaseSensitiveByDefault() throws Throwable
+    {
+        createTable("CREATE TABLE %s (id text PRIMARY KEY, val text)");
+
+        createIndex("CREATE CUSTOM INDEX ON %s(val) USING 'StorageAttachedIndex'");
+        waitForIndexQueryable();
```

Comment:
Since [CASSANDRA-18521](https://issues.apache.org/jira/browse/CASSANDRA-18521) `CQLTester#waitForIndexQueryable` requires an argument specifying the index name. Its absence here produces a build error. Maybe it has been missed during rebase?

Reply:
Note that also `CQLTester#waitForTableIndexesQueryable` can be used to just wait for all the indexes used by the table.

Reply:
Since `waitForTableIndexesQueryable` was added to `createIndex`, I have removed these calls.